lib/iris/tests/unit/analysis/cartography/test__get_lon_lat_coords.py

```diff
@@ -0,0 +1,114 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""Test function :func:`iris.analysis.cartography._get_lon_lat_coords"""
+
+import pytest
+
+from iris.analysis.cartography import _get_lon_lat_coords as g_lon_lat
+from iris.coords import AuxCoord
+from iris.tests.stock import lat_lon_cube
+
+
+@pytest.fixture
+def dim_only_cube():
+    yield lat_lon_cube()
```

Comment:
I believe this should work as just `return`. From what I can tell of pytest, `yield` tends to be used when there is teardown code to be run afterwards.

Reply:
I was originally under the impression that they were interchangable, in which case the one with the most potential functionality would've made most sense. In hindsight, they wouldn't have both for no reason (I'd hope) so changed it back!